docs/Userguide_login.rst

```diff
@@ -13,34 +13,139 @@ You can access the Mila cluster via ssh:
 
 .. prompt:: bash $
 
+    # Generic login, will send you to one of the 4 login nodes to spread the load
     ssh <user>@login.server.mila.quebec -p 2222
 
+    # To connect to a specific login node, X in [1, 2, 3, 4]
+    ssh <user>@login-X.login.server.mila.quebec -p 2222
+
 Four login nodes are available and accessible behind a load balancer. At each
-connection, you will be redirected to the least loaded login-node. Each login
-node can be directly accessed via: ``login-X.login.server.mila.quebec`` on port
-``2222``.
+connection, you will be redirected to the least loaded login-node.
+
+The RSA, DSA and ECDSA fingerprints for Mila's login nodes are:
+
+.. code-block:: bash
+
+    SHA256:baEGIa311fhnxBWsIZJ/zYhq2WfCttwyHRKzAb8zlp8 (ECDSA)
+    SHA256:XvukABPjV75guEgJX1rNxlDlaEg+IqQzUnPiGJ4VRMM (DSA)
+    SHA256:Xr0/JqV/+5DNguPfiN5hb8rSG+nBAcfVCJoSyrR0W0o (RSA)
+    SHA256:gfXZzaPiaYHcrPqzHvBi6v+BWRS/lXOS/zAjOKeoBJg (ED25519)
+
+
+.. important::
+    Login nodes are merely *entry points* to the cluster. They give you access
+    to the compute nodes and to the filesystem, but they are not meant to run
+    anything heavy. Do **not** run compute-heavy programs on these nodes,
```

Comment:
I think I would say _You should not run anything different then a text editor because otherwise you could bring them down, impeding cluster access for everyone._. But it might be to aggressive

Reply:
I mean, we don't want them to run VSCode on the login nodes, so even that might send mixed messages ;)

Reply:
True even if VSCode is more of an IDE than a text editor but users might not do the difference